### PR TITLE
Add my information about Git.

### PR DESCRIPTION
**Pull request by lisnyak-o**

### CODE REVIEW

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Looks great! But you make some changes in bottom of the file, maybe add a new line, you should not change other people code ;)
Снимок экрана 2020-10-13 в 8 48 15 PM

**Comment by lisnyak-o** [comment]
Thanks @miroslav-kolomiets , I decided conflict, where`re maked mistakes.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Now it's good, but you need to resolve conflict in your branch.

**Comment by lisnyak-o** [comment]
I merge base branch and pull new changes in this project for my fork.

File: README.md [hunk 1]
```diff
@@ -63,6 +75,7 @@ PS. Check the Kottans in wheeeeeaaatt.
 
 
 
+
 # Hi Everyone =)
```
Comment (Temu4) [inline README.md:79]: Please try to not change other people's code.

File: README.md [hunk 2]
```diff
@@ -1423,7 +1436,6 @@ I'm Kyrylo, I have always dreamed of programming and this a great opportunity fo
 ## What's up Kottans!
 My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
 ![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
----
```
Comment (Temu4) [inline README.md:1426]: As well as here ⬆️

**Review by Temu4 — CHANGES_REQUESTED** [review]

**Comment by lisnyak-o** [comment]
Guys, please see it`s right made ?

It was : 

Result :

File: README.md [hunk 3]
```diff
@@ -46,6 +46,19 @@ It was an explosive start to front-end learning! At first it was hard for me bec
...
+My name is [Oleg](https://github.com/lisnyak-o). So, meet my cat, called Richard ;)
+
+![richard](gif/richard.jpg "Richard")
+
+If you’ve questions about the tasks or something you can always contact me at :
+# [linkendIn](https://www.linkedin.com/in/oleg-lisnyak-a888a017a/) or [GitHub](https://github.com/lisnyak-o)
+## You’re welcome !
+Version Control with GIT - Completed !
+# Let’s start to next step!
+
```
Comment (miroslav-kolomiets) [inline README.md:61]: Simply, the PR should contain nothing but these lines.

File: README.md [hunk 4]
```diff
@@ -56,6 +69,7 @@ It was an explosive start to front-end learning! At first it was hard for me bec
 I'm [Max](https://github.com/maxpetruk) and this is Greyjoy, the kitten I'm hoping to adopt soon :smile:![my_kitten](gif/my_kitten.jpg)
 
 
+
```
Comment (miroslav-kolomiets) [inline README.md:72]: It shouldn't be.

File: README.md [hunk 5]
```diff
@@ -72,6 +86,7 @@ PS. Check the Kottans in wheeeeeaaatt.
 
 
 
+
```
Comment (miroslav-kolomiets) [inline README.md:89]: It shouldn't be too.

File: README.md [hunk 6]
```diff
@@ -1444,7 +1459,6 @@ I'm Kyrylo, I have always dreamed of programming and this a great opportunity fo
 ## What's up Kottans!
 My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
 ![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
----
```
Comment (miroslav-kolomiets) [inline README.md:1447]: It shouldn't be too.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]

**Comment by babayK0** [comment]
@lisnyak-o resolve conflicts please)

**Comment by lisnyak-o** [comment]
I leave empty line to the end for comment, but when I push my file it's line haven't, why  it's happend?

Screenshot_1
Screenshot_2

**Comment by Temu4** [comment]
Hey @lisnyak-o !
I still see a lot of code changes with you commits.

I hope you can do smth like these: 
1. Turn off auto-format settings in your code redactor/IDE.
2. Roll back to initial state of this branch (before your changes)
3. Pull the latest changes from origin repo
4. Add you code

If you need more info, please, fill free to touch me 😸

**Comment by lisnyak-o** [comment]
I hope everything worked out.
  What I did ?
1) git reset --hard HEAD
2) git fetch upstream
3) Added my own text at the end of the file, added a photo, then did => commit
4) git pull master
5) Solved the conflict. Accepted the current changes, and canceled the incoming (from the pull master).
6) git push

image
Why this line changed to the same line I don't know. I didn't touch her!

**Comment by lisnyak-o** [comment]
I'm already confused. Can I create a new PR? After all, I can't push a rollback via reset to the server.
And how then to solve this situation, I can not even imagine.

**Comment by lisnyak-o** [comment]
I created a new PR.

**Pull request closed without merging**